src/storekit.cs

```diff
@@ -350,7 +372,7 @@ interface SKPaymentTransaction {
 		string TransactionIdentifier { get; }
 
 #if !MONOMAC
-		[Availability (Deprecated = Platform.iOS_7_0, Message = "Use 'NSBundle.AppStoreReceiptUrl' instead.")]
+		[Availability (Deprecated = Platform.iOS_7_0, Message = "Use 'NSBundle.AppStoreReceiptUrl' instead.")] // NoWatch??
```

Comment:
Same as above - should this be `[NoWatch]`?

Reply:
yes, even more if it's deprecated
in such case it might be present but we don't want to encourage people to use it :)

Reply:
yes, same, use NoWatch, easier to add it later.